packages/default/scss/drawer/_layout.scss

```diff
@@ -0,0 +1,168 @@
+
+        /* Scrollbar styles for Chrome, Safari and Opera */
+        &::-webkit-scrollbar {
+            width: $drawer-scrollbar-width;
+        }
+
+        &::-webkit-scrollbar-thumb {
+            border-radius: $drawer-scrollbar-radius;
+        }
+    }
+
+    .k-drawer-container {
+        border-style: solid;
+        border-width: 0;
+        box-sizing: border-box;
+        overflow-x: hidden;
+        white-space: nowrap;
+
+        .k-drawer-items {
+            padding: 0;
+            width: 0;
+            transition: all .3s ease-out;
+            overflow-x: hidden;
+
+            ul {
+                padding: 0;
+                margin: 0;
+            }
+
+            .k-drawer-item {
+                display: block;
+                line-height: $drawer-item-line-height;
+                font-size: $drawer-item-font-size;
+                padding: $drawer-item-padding-y $drawer-item-padding-x;
+                list-style: none;
+
+                &.k-drawer-separator {
+                    padding: 0;
+                    height: 1px;
+                }
+            }
+
+            .k-icon {
+                display: inline-block;
+                font-size: $drawer-icon-size;
+                padding: 0 $drawer-item-padding-x;
+            }
+
+            .k-item-text {
+                display: inline-block;
+                padding: 0 $drawer-item-padding-x;
+                vertical-align: middle;
+            }
+        }
+
+        &.expanded {
+            border-right-width: $drawer-border-width;
+
+            .k-drawer-items {
+                width: $drawer-initial-width; /* Default width of sidebar */
+            }
+        }
+    }
+
+    .k-drawer-wrapper {
+        overflow-x: hidden;
+
+        &.mini-mode {
+            .k-drawer-container {
+                border-right-width: $drawer-border-width;
+            }
+
+            .k-drawer-items {
+                overflow-y: visible;
+                overflow-x: initial;
+                width: $drawer-mini-initial-width; /* Default width of mini sidebar */
+            }
+
+             &.expanded .k-drawer-items {
+                width: $drawer-initial-width; /* Default width of sidebar */
+            }
+        }
+
+        &.overlay {
+            .k-drawer-container {
+                position: fixed;
+                left: 0;
+                right: auto;
+                height: 100%;
+                z-index: 10002;
+            }
+
+            .k-drawer-items {
+                max-width: 80vw;
+            }
+        }
+
+
+        &.push {
+            display: flex;
+
+            .k-drawer-content {
+                flex: 1 1 auto;
+            }
+
+            .k-drawer-container {
+                position: relative;
+                flex: 0 0 auto;
+            }
+        }
+
+        .k-overlay {
+            display: none;
+        }
+
+        &.expanded  {
+            .k-overlay {
+                display: block;
```

Comment:
as far as I see, this selector will never work, and this style is assigned via JS. Thus, I'd suggest removing it